src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -218,6 +220,13 @@ static ProjectCodeRefactoringProvider.ExtensionInfo GetExtensionInfo(ExportCodeR
             return null;
         }
 
+        private static void CreateLogProperties(Dictionary<string, object?> map, object refactoringProvider)
+        {
+            var type = refactoringProvider.GetType();
+            var telemetryId = type.GetTelemetryId();
+            map["TelemetryId"] = telemetryId.ToString();
```

Comment:
nit: no need to ToString() here, it will be called when the object map is serialized.